CMakeLists.txt

```diff
@@ -392,6 +395,13 @@ IF(NOT OPENGL_BACKEND STREQUAL "None")
 		ENDIF()
 	ENDIF()
 
+	IF(SDL_DISABLE_CONTROLLER_API)
+		ADD_DEFINITIONS("-DSDL_DISABLE_CONTROLLER_API")
+	ENDIF()
+
+	IF(SDL_RESOLUTION_INDEPENDANCE)
+		ADD_DEFINITIONS("-DSDL_RESOLUTION_INDEPENDANCE")
+	ENDIF()
```

Comment:
You can add these to `cmake/cmake_config.h.in` instead, the cmakedefine stuff can simplify it.